TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -603,7 +648,7 @@ public void onError(Throwable t) {
 
     private void restartTango() {
         if (mParameterNode != null) mParameterNode.setPreferencesFromParameterServer();
-        updateSaveMapButton();
+        updateLoadAndSaveMapButtons();
```

Comment:
The function is already called in `onTangoStatus`. But it is not sufficient, the buttons also need to be updated in `onConfigurationChanged`.